ceph_deploy/install.py

```diff
@@ -28,8 +28,48 @@ def sanitize_args(args):
     return args
 
 
+def detect_components(args, distro):
+    """
+    Since the package split, now there are various different ceph components to
+    install like:
+
+    * ceph
+    * ceph-mon
+    * ceph-osd
+    * ceph-mds
+
+    This helper function should parse the args that may contain specifics about
+    these flags and return the default if none are passed in (which is, install
+    everything)
+    """
+    flags = {
+        'install_osd': 'ceph-osd',
+        'install_rgw': 'ceph-radosgw',
+        'install_mds': 'ceph-mds',
+        'install_mon': 'ceph-mon',
+        'install_ceph': 'ceph',
```

Comment:
Correct, the "ceph-mon", "ceph-osd", and "ceph-mds" packages will always pull in "ceph". I think of "ceph" as "files common to each of the three servers".

"ceph-radosgw" doesn't depend on "ceph", though, so it can stand alone from "ceph".